package.json

```diff
@@ -9,62 +9,37 @@
   "title": "WooCommerce Square",
   "version": "4.7.0",
   "homepage": "https://woocommerce.com/products/woocommerce-square/",
-  "main": "Gruntfile.js",
   "scripts": {
-    "build": "composer install --no-dev && NODE_ENV=production grunt && NODE_ENV=production npm run build:webpack && npm run archive",
+    "build": "composer install --no-dev && npm run build:webpack && npm run archive",
     "build:dev": "composer install && grunt && npm run build:webpack",
     "build:webpack": "rimraf build/* && wp-scripts build",
-    "build-watch:grunt": "grunt watch",
     "build-watch:webpack": "rimraf build/* && wp-scripts start",
-    "lint:js": "wp-scripts lint-js assets --ext js",
-    "lint:js:fix": "wp-scripts lint-js assets --ext js --fix",
+    "lint:js": "wp-scripts lint-js src/ --ext js",
+    "lint:js:fix": "wp-scripts lint-js src/ --ext js --fix",
     "phpcompat": "./vendor/bin/phpcs --standard=phpcs-compat.xml.dist -p .",
     "update-deps": "npm install -g rimraf && rimraf node_modules && rimraf npm-shrinkwrap.json && npm install && npm shrinkwrap --dev",
     "archive": "composer archive --file=$npm_package_name --format=zip",
     "postarchive": "rm -rf $npm_package_name && unzip $npm_package_name.zip -d $npm_package_name && rm $npm_package_name.zip && zip -r $npm_package_name.zip $npm_package_name && rm -rf $npm_package_name",
     "woorelease": "npm run build",
     "env": "wp-env",
-    "test:env:start": "wp-env start",
-    "test:env:stop": "wp-env stop",
-    "test:env:clean": "wp-env clean",
-    "test:env:destroy": "wp-env destroy",
-    "test:e2e-run": "npx playwright test --config=tests/e2e/config/playwright.config.js"
+    "env:start": "wp-env start",
+    "env:stop": "wp-env stop",
+    "env:clean": "wp-env clean",
+    "env:destroy": "wp-env destroy",
+    "test:e2e": "npx playwright test --config=tests/e2e/config/playwright.config.js"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.10",
-    "@babel/preset-env": "^7.12.11",
     "@playwright/test": "^1.44.1",
-    "@woocommerce/dependency-extraction-webpack-plugin": "^1.4.0",
+    "@woocommerce/dependency-extraction-webpack-plugin": "^3.0.1",
     "@woocommerce/eslint-plugin": "^2.2.0",
     "@wordpress/env": "^10.0.0",
     "@wordpress/scripts": "^27.9.0",
```

Comment:
Looks like the most recent version is 28.2.0 so might be nice to bump that up as part of this work